src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,106 @@
+package wolox.training.models;
+
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class Book<Subtitle> {
+    @Id
```

Comment:
I see that you are trying to use a derived primary  key but here we need to have a simple primary key. Check [this](https://www.baeldung.com/jpa-entities)